test/packet/scripts/add_vagrant_box.sh

```diff
@@ -1,55 +1,174 @@
 #!/bin/bash
 
-if [[ $# -ne 1 ]]; then
-    echo "usage: add-vagrant-boxes.sh [vagrant_box_defaults.rb path]"
-    exit 1
+set -e
+
+check_cmd() {
+    for cmd in $@ ; do
+        if !(command -v $cmd >/dev/null) ; then
+            echo "Error: $cmd not found."
+            exit 1
+        fi
+    done
+}
+check_cmd curl vagrant
+
+usage() {
+    echo -e "usage: add-vagrant-boxes.sh [options] [vagrant_box_defaults.rb path]"
```

Comment:
`s/add-vagrant-boxes/add-vagrant-box` (error left over from old comment).

Reply:
Thanks I'll fix that.